synapse/handlers/register.py

```diff
@@ -116,7 +116,9 @@ def __init__(self, hs: "HomeServer"):
             self.pusher_pool = hs.get_pusherpool()
 
         self.session_lifetime = hs.config.registration.session_lifetime
-        self.access_token_lifetime = hs.config.registration.access_token_lifetime
+        self.access_token_lifetime = (
```

Comment:
Should we rename the property on `RegistrationHandler` to match?

(Same goes for `RefreshTokenServlet` below.)